docs/modules/data-structures/pages/cpmap.adoc

```diff
@@ -2,14 +2,26 @@
 :page-enterprise: true
 [[cpmap]]
 
+Hazelcast `CPMap` is a distributed implementation of a minimal key-value interface.
+It is a CP data structure, so unlike the traditional maps built for availability (AP),
+`CPMap` guarantees data consistency even during network partitions, ensuring that every cluster member always sees the same data.
+This reliability makes it ideal for storing sensitive information such as user bank balances and system configurations, or managing mission-critical workflows.
 
-Hazelcast `CPMap` is a distributed implementation of a minimal key-value interface. It supports 
-atomic `compareAndSet`, `delete`, `get`, `put`, `putIfAbsent`, `remove` and `set` operations. As 
-`CPMap` is a distributed key-value implementation, these operations involve remote calls and their 
-performance differs from non-distributed key-value data structures; for example, `java.util.HashMap`. 
-`CPMap` is only available in the Enterprise Edition.
+`CPMap` data structure supports the following atomic operations:
 
-NOTE: Your license must include `ADVANCED_CP` to activate this feature and you must use the
+* `compareAndSet`
+* `delete`
+* `get`
+* `put`
+* `putIfAbsent`
+* `remove`
+* `set`
+
+As it is a distributed key-value implementation, above operations involve remote calls, and their 
+performance differs from non-distributed structures like ``HashMap``s.
```

Comment:
how does it differ?

Reply:
Wanna say slower? : P

Reply:
@rebekah-lawrence WDYT about

"Due to CPMap's distributed nature, the above operations involve remote calls and might not be as speedy as they are with non-distributed counterparts such as HashMaps."

Reply:
Due to CPMap's distributed nature, the operations involve remote calls. This means that the calls might take longer to complete than non-distributed counterparts, such as HashMaps.

?